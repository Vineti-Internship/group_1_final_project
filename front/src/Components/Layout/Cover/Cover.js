import React from 'react'
import './Cover.css';

class Cover extends React.Component {
  render() {
    return (
        <div>
         <div class="cover-container d-flex w-100 h-100 p-3 mx-auto flex-column">
        <header class="masthead mb-auto">
        <div class="inner">
          <h3 class="masthead-brand">WeWork</h3>
          <nav class="nav nav-masthead justify-content-center">
            <a class="nav-link" href="#">Sign up</a>
            <a class="nav-link" href="/signin">Sign in</a>
          </nav>
        </div>
      </header>
      </div>
      <div class = "Cover"> 
        <main role="main" class=" inner cover">
            <h1 class="cover-heading">Welcome to WeWork</h1>
            <p class="lead">WeWork is a full service, creative working environment with a unique entrepreneurial spirit.</p>
            <p class="lead">
            <a href="/about" class="btn btn-lg btn-secondary">Learn more</a>
            </p>
        </main>
      </div>
      </div>
    )
  }
}

export default Cover;