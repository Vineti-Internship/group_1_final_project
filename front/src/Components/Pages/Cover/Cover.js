import React from 'react'

function Cover() {
  return (
    <div>
    <div className="cover-container d-flex w-100 h-100 p-3 mx-auto flex-column gray-navbar">
      <header className="masthead mb-auto container">
        <div className="inner container">
          <a className="navbar-brand nav-main" href="/">WeWork</a>
          <nav className="nav nav-masthead justify-content-center">
            <a className="nav-link" href="/signup">Sign up</a>
            <a className="nav-link" href="/signin">Sign in</a>
          </nav>
        </div>
      </header>
    </div>
    <div className = "Custom lower-height-cover"> 
      <main role="main" className=" inner cover">
          <h1 className="cover-heading">Welcome to WeWork</h1>
          <p className="lead">WeWork is a full service, creative working environment with a unique entrepreneurial spirit.</p>
          <p className="lead">
            <a href="/about" className="btn btn-lg btn-outline-light">Learn more</a>
          </p>
      </main>
    </div>
  </div>
  )
}

export default Cover;