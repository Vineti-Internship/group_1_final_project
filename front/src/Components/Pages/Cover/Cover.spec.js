import React from "react";
import { shallow } from "enzyme";
import Cover from "./Cover";

describe("Cover", () => {

    it("does not change over time", () => {
        const wrapper = shallow(
            <Cover />
        );
        expect(wrapper).toMatchSnapshot();
    })

});