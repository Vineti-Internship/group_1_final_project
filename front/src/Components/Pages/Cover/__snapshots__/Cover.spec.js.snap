// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cover does not change over time 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Cover />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="cover-container d-flex w-100 h-100 p-3 mx-auto flex-column gray-navbar"
        >
          <header
            className="masthead mb-auto container"
          >
            <div
              className="inner container"
            >
              <a
                class="navbar-brand nav-main"
                href="/"
              >
                WeWork
              </a>
              <nav
                className="nav nav-masthead justify-content-center"
              >
                <a
                  className="nav-link"
                  href="/signup"
                >
                  Sign up
                </a>
                <a
                  className="nav-link"
                  href="/signin"
                >
                  Sign in
                </a>
              </nav>
            </div>
          </header>
        </div>,
        <div
          className="Custom lower-height-cover"
        >
          <main
            className=" inner cover"
            role="main"
          >
            <h1
              className="cover-heading"
            >
              Welcome to WeWork
            </h1>
            <p
              className="lead"
            >
              WeWork is a full service, creative working environment with a unique entrepreneurial spirit.
            </p>
            <p
              className="lead"
            >
              <a
                className="btn btn-lg btn-outline-light"
                href="/about"
              >
                Learn more
              </a>
            </p>
          </main>
        </div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <header
            className="masthead mb-auto container"
          >
            <div
              className="inner container"
            >
              <a
                class="navbar-brand nav-main"
                href="/"
              >
                WeWork
              </a>
              <nav
                className="nav nav-masthead justify-content-center"
              >
                <a
                  className="nav-link"
                  href="/signup"
                >
                  Sign up
                </a>
                <a
                  className="nav-link"
                  href="/signin"
                >
                  Sign in
                </a>
              </nav>
            </div>
          </header>,
          "className": "cover-container d-flex w-100 h-100 p-3 mx-auto flex-column gray-navbar",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="inner container"
            >
              <a
                class="navbar-brand nav-main"
                href="/"
              >
                WeWork
              </a>
              <nav
                className="nav nav-masthead justify-content-center"
              >
                <a
                  className="nav-link"
                  href="/signup"
                >
                  Sign up
                </a>
                <a
                  className="nav-link"
                  href="/signin"
                >
                  Sign in
                </a>
              </nav>
            </div>,
            "className": "masthead mb-auto container",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <a
                  class="navbar-brand nav-main"
                  href="/"
                >
                  WeWork
                </a>,
                <nav
                  className="nav nav-masthead justify-content-center"
                >
                  <a
                    className="nav-link"
                    href="/signup"
                  >
                    Sign up
                  </a>
                  <a
                    className="nav-link"
                    href="/signin"
                  >
                    Sign in
                  </a>
                </nav>,
              ],
              "className": "inner container",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "WeWork",
                  "class": "navbar-brand nav-main",
                  "href": "/",
                },
                "ref": null,
                "rendered": "WeWork",
                "type": "a",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <a
                      className="nav-link"
                      href="/signup"
                    >
                      Sign up
                    </a>,
                    <a
                      className="nav-link"
                      href="/signin"
                    >
                      Sign in
                    </a>,
                  ],
                  "className": "nav nav-masthead justify-content-center",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Sign up",
                      "className": "nav-link",
                      "href": "/signup",
                    },
                    "ref": null,
                    "rendered": "Sign up",
                    "type": "a",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Sign in",
                      "className": "nav-link",
                      "href": "/signin",
                    },
                    "ref": null,
                    "rendered": "Sign in",
                    "type": "a",
                  },
                ],
                "type": "nav",
              },
            ],
            "type": "div",
          },
          "type": "header",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <main
            className=" inner cover"
            role="main"
          >
            <h1
              className="cover-heading"
            >
              Welcome to WeWork
            </h1>
            <p
              className="lead"
            >
              WeWork is a full service, creative working environment with a unique entrepreneurial spirit.
            </p>
            <p
              className="lead"
            >
              <a
                className="btn btn-lg btn-outline-light"
                href="/about"
              >
                Learn more
              </a>
            </p>
          </main>,
          "className": "Custom lower-height-cover",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1
                className="cover-heading"
              >
                Welcome to WeWork
              </h1>,
              <p
                className="lead"
              >
                WeWork is a full service, creative working environment with a unique entrepreneurial spirit.
              </p>,
              <p
                className="lead"
              >
                <a
                  className="btn btn-lg btn-outline-light"
                  href="/about"
                >
                  Learn more
                </a>
              </p>,
            ],
            "className": " inner cover",
            "role": "main",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Welcome to WeWork",
                "className": "cover-heading",
              },
              "ref": null,
              "rendered": "Welcome to WeWork",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "WeWork is a full service, creative working environment with a unique entrepreneurial spirit.",
                "className": "lead",
              },
              "ref": null,
              "rendered": "WeWork is a full service, creative working environment with a unique entrepreneurial spirit.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  className="btn btn-lg btn-outline-light"
                  href="/about"
                >
                  Learn more
                </a>,
                "className": "lead",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Learn more",
                  "className": "btn btn-lg btn-outline-light",
                  "href": "/about",
                },
                "ref": null,
                "rendered": "Learn more",
                "type": "a",
              },
              "type": "p",
            },
          ],
          "type": "main",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="cover-container d-flex w-100 h-100 p-3 mx-auto flex-column gray-navbar"
          >
            <header
              className="masthead mb-auto container"
            >
              <div
                className="inner container"
              >
                <a
                  class="navbar-brand nav-main"
                  href="/"
                >
                  WeWork
                </a>
                <nav
                  className="nav nav-masthead justify-content-center"
                >
                  <a
                    className="nav-link"
                    href="/signup"
                  >
                    Sign up
                  </a>
                  <a
                    className="nav-link"
                    href="/signin"
                  >
                    Sign in
                  </a>
                </nav>
              </div>
            </header>
          </div>,
          <div
            className="Custom lower-height-cover"
          >
            <main
              className=" inner cover"
              role="main"
            >
              <h1
                className="cover-heading"
              >
                Welcome to WeWork
              </h1>
              <p
                className="lead"
              >
                WeWork is a full service, creative working environment with a unique entrepreneurial spirit.
              </p>
              <p
                className="lead"
              >
                <a
                  className="btn btn-lg btn-outline-light"
                  href="/about"
                >
                  Learn more
                </a>
              </p>
            </main>
          </div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <header
              className="masthead mb-auto container"
            >
              <div
                className="inner container"
              >
                <a
                  class="navbar-brand nav-main"
                  href="/"
                >
                  WeWork
                </a>
                <nav
                  className="nav nav-masthead justify-content-center"
                >
                  <a
                    className="nav-link"
                    href="/signup"
                  >
                    Sign up
                  </a>
                  <a
                    className="nav-link"
                    href="/signin"
                  >
                    Sign in
                  </a>
                </nav>
              </div>
            </header>,
            "className": "cover-container d-flex w-100 h-100 p-3 mx-auto flex-column gray-navbar",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="inner container"
              >
                <a
                  class="navbar-brand nav-main"
                  href="/"
                >
                  WeWork
                </a>
                <nav
                  className="nav nav-masthead justify-content-center"
                >
                  <a
                    className="nav-link"
                    href="/signup"
                  >
                    Sign up
                  </a>
                  <a
                    className="nav-link"
                    href="/signin"
                  >
                    Sign in
                  </a>
                </nav>
              </div>,
              "className": "masthead mb-auto container",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <a
                    class="navbar-brand nav-main"
                    href="/"
                  >
                    WeWork
                  </a>,
                  <nav
                    className="nav nav-masthead justify-content-center"
                  >
                    <a
                      className="nav-link"
                      href="/signup"
                    >
                      Sign up
                    </a>
                    <a
                      className="nav-link"
                      href="/signin"
                    >
                      Sign in
                    </a>
                  </nav>,
                ],
                "className": "inner container",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "WeWork",
                    "class": "navbar-brand nav-main",
                    "href": "/",
                  },
                  "ref": null,
                  "rendered": "WeWork",
                  "type": "a",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <a
                        className="nav-link"
                        href="/signup"
                      >
                        Sign up
                      </a>,
                      <a
                        className="nav-link"
                        href="/signin"
                      >
                        Sign in
                      </a>,
                    ],
                    "className": "nav nav-masthead justify-content-center",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Sign up",
                        "className": "nav-link",
                        "href": "/signup",
                      },
                      "ref": null,
                      "rendered": "Sign up",
                      "type": "a",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Sign in",
                        "className": "nav-link",
                        "href": "/signin",
                      },
                      "ref": null,
                      "rendered": "Sign in",
                      "type": "a",
                    },
                  ],
                  "type": "nav",
                },
              ],
              "type": "div",
            },
            "type": "header",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <main
              className=" inner cover"
              role="main"
            >
              <h1
                className="cover-heading"
              >
                Welcome to WeWork
              </h1>
              <p
                className="lead"
              >
                WeWork is a full service, creative working environment with a unique entrepreneurial spirit.
              </p>
              <p
                className="lead"
              >
                <a
                  className="btn btn-lg btn-outline-light"
                  href="/about"
                >
                  Learn more
                </a>
              </p>
            </main>,
            "className": "Custom lower-height-cover",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h1
                  className="cover-heading"
                >
                  Welcome to WeWork
                </h1>,
                <p
                  className="lead"
                >
                  WeWork is a full service, creative working environment with a unique entrepreneurial spirit.
                </p>,
                <p
                  className="lead"
                >
                  <a
                    className="btn btn-lg btn-outline-light"
                    href="/about"
                  >
                    Learn more
                  </a>
                </p>,
              ],
              "className": " inner cover",
              "role": "main",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Welcome to WeWork",
                  "className": "cover-heading",
                },
                "ref": null,
                "rendered": "Welcome to WeWork",
                "type": "h1",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "WeWork is a full service, creative working environment with a unique entrepreneurial spirit.",
                  "className": "lead",
                },
                "ref": null,
                "rendered": "WeWork is a full service, creative working environment with a unique entrepreneurial spirit.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    className="btn btn-lg btn-outline-light"
                    href="/about"
                  >
                    Learn more
                  </a>,
                  "className": "lead",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Learn more",
                    "className": "btn btn-lg btn-outline-light",
                    "href": "/about",
                  },
                  "ref": null,
                  "rendered": "Learn more",
                  "type": "a",
                },
                "type": "p",
              },
            ],
            "type": "main",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
